CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.11.2] - released on 2017-04-04
+### Added
+- Added the `link_target` keyword argument to `install_service`. If `link_target` is specified, instead of installing the `service_file` in the platform/init system default for service files, the file will be installed to the `link_target` and a link will be added from the system default for service files to the link target.
```

Comment:
`will be added from the system's default location for service files to the link target`?